strawberry/subscriptions/protocols/graphql_ws/handlers.py

```diff
@@ -97,7 +97,9 @@ async def handle_message(
 
     async def handle_connection_init(self, message: ConnectionInitMessage) -> None:
         payload = message.get("payload")
-        if payload is not None and not isinstance(payload, dict):
+        if payload is None:
```

Comment:
For graphql_ws before this PR connection_params could be null, if payload is null.
So this may be a slightly breaking change.